stix_shifter_modules/carbonblack/configuration/config.json

```diff
@@ -1,7 +1,8 @@
 {
     "connection": {
         "type": {
-            "default": "CarbonBlack"
+            "default": "carbonblack",
+            "displayName": "Carbon Black CB Response"
```

Comment:
"displayName": "Carbon Black Response"

I know `CB` is there in the existing display name but I don't understand why its there as we already mentioned `Carbon Black`